### ISSUE TITLE
ACCESSIBILITY REMEDIATION: Link Advanced Search Directions to Form

### ISSUE THREAD

**Issue by dl-maura** [yalelibrary/YUL-DC#779]
For screen reader users, we want to programmatically tie together the Advanced Search element and the Search Tips instructions. Visual users can see the notes and easily read through them before searching. We can create a similar experience for screen reader users by associating the notes to the input field using aria-describedby.

**ACCEPTANCE**
- [x] Each *input field* on the Advanced Search form has an `aria-describedby` pointing at the "Advanced Search Tips" (NOTE: you'll have to add an ID at the level of the `adv-search-help` or `adv-search-help.card-body` level)
   - [x] "Find Items that Match"
   - [x] "Sort Results By"
   - [x] "All Fields"
   - [x] "Creator"
   - [x] "Title"
   - [x] "Call Number"
   - [x] "Date"
   - [x] "Subject"
   - [x] "Genre/format"
   - [x] "OID [Parent/primary]"
   - [x] "OID [Child/images]"

NOTE: `aria-describedby` should be applied to the input fields themselves and not the associated labels.  See W3C [example 2](https://www.w3.org/TR/WCAG20-TECHS/ARIA1.html#ARIA1-examples) for an example similar to our use case.

**REFERENCES**
* W3C [ARIA1: Using the aria-describedby property to provide a descriptive label for user interface controls](https://www.w3.org/TR/WCAG20-TECHS/ARIA1.html) - see especially example 2
* Mozilla Developer Network [Using the aria-describedby attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute)

**Comment by dl-maura** [comment]
From Kara:
For screen reader users, we want to programmatically tie together the Advanced Search element and the Search Tips instructions. Visual users can see the notes and easily read through them before searching. We can create a similar experience for screen reader users by associating the notes to the input field using aria-describedby.
 
Using aria-describedby (Example 2)

**Comment by martinlovell** [comment]
Some info on where to put of overall instructions:   https://www.w3.org/WAI/tutorials/forms/instructions/

**Comment by maxkadel** [comment]
![image](https://user-images.githubusercontent.com/45948126/98847073-c7d1d080-241d-11eb-85de-47db352a20e1.png)
